live/gov/account.hcl

```diff
@@ -1,6 +1,6 @@
 # Set account-wide variables. These are automatically pulled in to configure the remote state bucket in the root
 # terragrunt.hcl configuration.
 locals {
-  aws_account_id = "<ENTER AWS GOVCLOUD ACCT #>"
+  aws_account_id = "446787640263"
```

Comment:
Is this an intended change?

Reply:
Yes, remember we discussed the fact that our standard and govcloud AWS ID was already available publicly (necessarily for access to our Lambda App) in our CloudFormation so it didn't make a lot of sense to censor it here. Including it for our testing platform adds a lot of convenience.

Reply:
It's not entirely clear to me how valuable an AWS account ID is. It does seem like this lowers the barrier to collecting information compared to other locations. An example that comes to mind is a search like `aws_account_id site:github.com`.

I don't know enough about AWS account security to make a call either way on this but wanted to raise it as a potential concern just in case.

Reply:
My point it is doesn't really matter since we have a hard requirement to have the AWS account ID public for our Lambda's to work, see here: https://github.com/Dozuki/CloudPrem/blob/add32e387eb41f69e1524c6753d835574e601fcc/codepipeline_app.yml#L59